### DISCUSSION TITLE
Add editor to mobile app

### DISCUSSION THREAD

**Discussion by ghost** [community/community#61035] [Mobile]
Please add editor and options(e.g. fork and repo settings) to the mobile app and the mobile webpage. I need this to help translating some projects.

**Comment by esahmed21** [comment]
Hi @LiuHaoming1145 👋

Thank you for reaching out to us!
- For file editing you can now Edit files and also create PRs from the mobile app, check this [blog](https://github.blog/changelog/2022-11-15-github-for-ios-edit-files-from-browse-code) for more details about the feature. 🎊
- For the repository settings, I'll make sure to create an issue to track it and discuss internally with the team!

Let me know if there is something else I can help with.
Thank you!

**Reply by ghost** [reply]
I need Android app!

**Reply by ghost** [reply]
Why not add download link on github instead of google play?

**Reply by esahmed21** [reply]
Hi @LiuHaoming1145 👋

The Editing Files feature exists also on Android platform. It works exactly the same way.

We will make sure to update and add links for Android.

Thanks for reaching out to us!

**Reply by ghost** [reply]
Thanks pls add more options like fork on mobile(app/website). I can only use my phone because I have no time to use computer.

**Reply by esahmed21** [reply]
Hi @LiuHaoming1145 👋

I'll pass your request to the team and create issue internally to track it and discuss with the team.

Thank you!

**Comment by Antonio12389** [comment]
> Please add editor and options(e.g. fork and repo settings) to the mobile app and the mobile webpage. I need this to help translating some projects.

Me pueden ayudar por favor

**Comment by Antonio12389** [comment]
> Please add editor and options(e.g. fork and repo settings) to the mobile app and the mobile webpage. I need this to help translating some projects.

Me pueden ayudar por favor

**Reply by CryptoHeni793** [reply]
@

**Comment by CryptoHeni793** [comment]
> Please add editor and options(e.g. fork and repo settings) to the mobile app and the mobile webpage. I need this to help translating some projects.

**Reply by CryptoHeni793** [reply]
J
